src/main/java/com/google/api/codegen/viewmodel/testing/MockCombinedView.java

```diff
@@ -30,7 +30,9 @@
 
   public abstract List<MockServiceImplView> serviceImpls();
 
-  // TODO(pongad): Add the tests
+  public abstract List<GapicSurfaceTestClassView> tests();
```

Comment:
I think `testClasses` (or `testFiles`) would be better, because `tests` is ambiguous with `testCases`